src/libexec/certbot

```diff
@@ -5,16 +5,24 @@ if [ "$(whoami)" != "zextras" ]; then
   exit 1
 fi
 
+delete_command="certbot delete -n --cert-name"
+
 certonly() {
   if certbot "${@}" --dry-run; then
     certbot "${@}"
   fi
 }
 
+delete() {
+  for domain in "${@}"; do
+    ${delete_command} "$domain"
+  done
+}
+
 ACTION="${1}"
 
 if [ "${ACTION}" = "certonly" ]; then
   certonly "${@}"
+elif [ "${ACTION}" = "delete" ]; then
```

Comment:
IMHO a case switch would be better, something like
```
case "$1" in
certonly)
  certonly "${@}"
  ;;
delete)
  delete "${@:2}"
  ;;
*)
  echo "Usage: $0 certonly|delete"
  exit 1
  ;;
esac
```
avoiding unneeded usage of a global variable (ACTION)